PrintingPodRecharge/ModAssets.cs

```diff
@@ -60,11 +60,11 @@ public static void LateLoadAssets()
 
 			Prefabs.bioInkSideScreen = bundle.LoadAsset<GameObject>("Assets/UIs/BioInkSidescreen.prefab");
 			Log.Assert("sidescreen", Prefabs.bioInkSideScreen);
-			tmp.ReplaceAllText(Prefabs.bioInkSideScreen);
+			TMPConverter.ReplaceAllText(Prefabs.bioInkSideScreen);
 
 			Prefabs.settingsDialog = bundle.LoadAsset<GameObject>("Assets/UIs/SettingsDialog 1.prefab");
 			Log.Assert("settingsDialog", Prefabs.settingsDialog);
-			tmp.ReplaceAllText(Prefabs.settingsDialog);
+			TMPConverter.ReplaceAllText(Prefabs.settingsDialog);
```

Comment:
This was a compile time error (possibly either .NET or C# version changes caused this to be not compilable anymore), but no meaningful change in functionality. The complaint was technically correct, but I do not know why it was a compile time error, it was CS0176